lib/ts/recipe/dashboard/api/listTenants.ts

```diff
@@ -39,7 +40,7 @@ export default async function listTenants(
     _: APIInterface,
     __: string,
     ___: APIOptions,
-    userContext: any
+    userContext: UserContext
 ): Promise<Response> {
     let tenantsRes = await Multitenancy.listAllTenants(userContext);
     let finalTenants: TenantListTenantType[] = [];
```

Comment:
this API also needs to change cause we now have firstFactors, requiredSecondaryFactors etc also a part of the tenant config.

Reply:
Done